site/docs/v1/tech/reactor.adoc

```diff
@@ -27,8 +27,7 @@ Then in your FusionAuth instance navigate to [breadcrumb]#Reactor# and enter you
 _Note: You may enter your License Id on multiple instances.
 However, since billing is based upon the total number of monthly active users across all licensed instances it is recommended that you only license those instances you need support for or require Reactor features for._
```

Comment:
Since you are in here, I'd mention that we provide a developer license key as well, which is suitable for development, QA or UAT instances.

Reply:
I'd also change this from emphasis to 

```
[NOTE]
====
You may enter your License Id on multiple instances.

However, since billing is based upon the total number of monthly active users across all licensed instances it is recommended that you only license those instances you need support for or require Reactor features for.
====
```

As that is more in keeping with how we do callouts.